lib/presentation/pages/layouts/profile/profile_avatar_image.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:flutter/material.dart';
+
+class ProfileAvatarImage extends StatelessWidget {
+  const ProfileAvatarImage({
+    super.key,
+    required this.url,
+  });
+  final String url;
+  @override
+  Widget build(BuildContext context) {
+    return ClipOval(
+      child: Image.network(
```

Comment:
If possible write a custom NetworkImage widget which will handle failure case, image caching, loading placeholder e.t.c Not a priority as of now